packages/expo-modules-core/android/src/main/cpp/MethodMetadata.h

```diff
@@ -74,16 +74,6 @@ class MethodMetadata {
 
   MethodMetadata(MethodMetadata &&other) = default;
 
-  /**
-   * MethodMetadata owns the only reference to the Kotlin function.
-   * We have to clean that, cause it's a `global_ref`.
-   */
-  ~MethodMetadata() {
-    if (jBodyReference != nullptr) {
-      jBodyReference.release();
```

Comment:
`jni::global_ref.release` doesn't release the reference. It releases the ownership. So it was leaking. To avoid such problems in the future, we shouldn't override destructors when we don't need to 🤦‍♂️ 
Memory will be deallocated when the `jni::global_ref` destructor is called.